src/main/java/ch/njol/skript/expressions/ExprMaxHealth.java

```diff
@@ -66,13 +66,13 @@ protected String getPropertyName() {
 		return "max health";
 	}
 	
-	@Override
-	@Nullable
-	public Class<?>[] acceptChange(final ChangeMode mode) {
-		if(!Skript.isRunningMinecraft(1, 5, 2)) {
-			Skript.error("The max health of an entity can only be changed in Minecraft 1.6 and later");
-			return null;
-		}
+	@Override
+	@Nullable
+	public Class<?>[] acceptChange(final ChangeMode mode) {
+		if(!Skript.isRunningMinecraft(1, 5, 2)) {

```

Comment:
Why these lines look like changed?  can't see the difference but
```suggestion
		if (!Skript.isRunningMinecraft(1, 5, 2)) {
```
Actually this check should be removed..